tests/runner.py

```diff
@@ -524,8 +524,17 @@ def get_dir(self):
   def in_dir(self, *pathelems):
     return os.path.join(self.get_dir(), *pathelems)
 
-  def get_stdout_path(self):
-    return os.path.join(self.get_dir(), 'stdout')
+  def add_pre_run(self, code):
+    create_test_file('pre.js', 'Module.preRun = function() { %s }' % code)
```

Comment:
it worries me that these three all have the same `pre.js` filename. right now only one of the three is possible per test I think? Using a different name for each would be safer.

Reply:
Sure.. good idea.